### DISCUSSION TITLE
Deprecating the aws-prototyping-sdk bundle/uber package

### DISCUSSION THREAD

**Discussion by agdimech** [aws/aws-pdk#218] [Polls]
Hi,

As part of the PDK we vend `aws-prototyping-sdk` which bundles all STABLE packages into a single distributable. Based on npm metrics, it appears most people actually favour consuming individual packages from the `@aws-prototyping-sdk` namespace and has led me to create this poll to understand if this deduction is accurate and if the community would support the removal of this uber package as it will simplify the codebase substantially.

**Comment by JeremyJonas** [comment]
We will be deprecating the `aws-prototyping-sdk` bundled package, in favor of only publishing the individual packages under `@aws-prototyping-sdk/*` scope, such as `@aws-prototyping-sdk/nx-monorepo`.

Please start using the `@aws-prototyping-sdk/` scoped packages ASAP. Thank you.
